src/content/api.js

```diff
@@ -64,26 +64,35 @@ function getEntitiesFromWebpage(webpageUrl) {
 }
 
 // query Dandelion for all unique locations from the text on the current page
-export function getUniqueLocationsFromCurrentPage() {
-    const currentPageUrl = getCurrentPageUrl();
-    return new Promise((resolve, reject) => {
-        getEntitiesFromWebpage(currentPageUrl)
-            .then(JSON.parse)
-            .then(
-                (response) => {
-                    // filter the reponse for all entities that are locations
-                    // then remove duplicate locations... ones that have the same "spot"
-                    resolve(filterDuplicates(filterLocations(response), 'spot'));
-                },
-                (error) => {
-                    alert(
-                        'Error: API.JS \n--------------\n Could not get entities from webpage \n---------------\n',
-                    );
-                    reject(error);
-                },
-            );
-    });
-}
+export const getUniqueLocationsFromCurrentPage = () => new Promise((resolve) => {
```

Comment:
dont need the {} here... you don't have a function state

Reply:
What do you mean?

Reply:
No brackets needed after (resolve) =>
Rather...
```
export const getUniqueLocationsFromCurrentPage = () => new Promise(resolve => ...)
```